### ISSUE TITLE
Global banner missing role attribute

### ISSUE THREAD

**Issue by luketa8** [grommet/hpe-design-system#4360]
### What does this request relate to?

Accessibility

### Describe the feature you'd like to request.

From [Guidance](https://www.w3.org/TR/WCAG20-TECHS/ARIA11.html):

Landmark roles (or "landmarks") programmatically identify sections of a page. Landmarks help assistive technology (AT) users orient themselves to a page and help them navigate easily to various sections of a page.

Landmarks should have a unique role or role/label/title (i.e. accessible name) combination.

The `role=”banner”` ARIA landmark should only appear once on an element that is unique to the HTML document

- Add a role attribute to the Global Header

### How urgent is this request?

None
